usecase/usecase-webapi-xwalk-tests/testscripts/steps/steps.py

```diff
@@ -47,6 +47,14 @@ def check_type_by_key(context, key, expecttype):
 def check_type_by_key(context, p_name, key):
     assert context.app.save_content(p_name, key)
 
+@step(u'I commit message by click "{key}"')
```

Comment:
This step name elusive, here you want to click element by id with js method. Please update it.